plugins/bazaar-backend/src/service/router.ts

```diff
@@ -51,12 +51,20 @@ export async function createRouter(
     }
   });
 
-  router.put('/projects/:id/member/:userId', async (request, response) => {
-    const { id, userId } = request.params;
-    await dbHandler.addMember(parseInt(id, 10), userId, request.body?.picture);
+  router.put(
+    '/projects/:id/member/:userId/:userRef?',
```

Comment:
I think it's best to avoid passing `userRef` as a path param. It could be passed through a query param or in the body, but I think the best would be to grab it from the user token once #12542 has landed. I would say it's worth waiting for that.

Reply:
Ack! That's much better!

Reply:
Heads up that #12542 has now landed, and even a couple of extra docs in #13482